packages/synapse-react-client/src/lib/containers/download_list_v2/DownloadListActionsRequired.tsx

```diff
@@ -12,6 +15,48 @@ export type DownloadListActionsRequiredProps = {
   onViewSharingSettingsClicked: (benefactorId: string) => void
 }
 
+/**
+ * Returns rendering for the Action and optional count.
+ *
+ * @param {Action} action action being rendered
+ * @param {number} count number of files associated to the Action (optional)
+ */
+export const renderActionRequired = (action: Action, count?: number) => {
```

Comment:
Existing code, refactored for reuse (and break apart Action and count to make count optional)